pallets/ddc-staking/src/lib.rs

```diff
@@ -69,6 +73,39 @@ parameter_types! {
 	pub MaxUnlockingChunks: u32 = 32;
 }
 
+/// Reward points of an era. Used to split era total payout between stakers.
+#[derive(PartialEq, Encode, Decode, RuntimeDebug, TypeInfo, Clone)]
+pub struct EraRewardPoints<AccountId: Ord> {
+	/// Total number of points. Equals the sum of reward points for each staker.
+	pub total: RewardPoint,
+	/// The reward points earned by a given staker.
+	pub individual: BTreeMap<AccountId, RewardPoint>,
+}
+
+/// Reward points for particular era. To be used in a mapping.
+#[derive(PartialEq, Encode, Decode, RuntimeDebug, TypeInfo, Clone)]
+pub struct EraRewardPointsPerNode {
+	/// Era points accrued
+	pub era: EraIndex,
+	/// Total number of points for node
+	pub points: RewardPoint,
+}
+
+/// Reward paid for some era.
+#[derive(PartialEq, Encode, Decode, RuntimeDebug, TypeInfo, Clone)]
+pub struct EraRewardsPaid<Balance: HasCompact> {
```

Comment:
Why do we store paid rewards?

Reply:
no indexer available for cluster management currently. This should be marked as obsolete for mainnet